src/ocrd_utils/os.py

```diff
@@ -75,27 +75,38 @@ def unzip_file_to_dir(path_to_zip, output_directory):
     z.close()
 
 @lru_cache()
-def get_ocrd_tool_json(executable):
+def get_ocrd_tool_json(executable, skip_error_print: bool = False):
     """
     Get the ``ocrd-tool`` description of ``executable``.
     """
+    ocrd_tool = {}
     executable_name = Path(executable).name
-    try:
-        ocrd_tool = loads(run([executable, '--dump-json'], stdout=PIPE).stdout)
-    except (JSONDecodeError, OSError) as e:
-        getLogger('ocrd.utils.get_ocrd_tool_json').error(f'{executable} --dump-json produced invalid JSON: {e}')
-        ocrd_tool = {}
+    # TODO: Potentially these executables should be moved as a constant somewhere else
+    skip_executables = ["ocrd-cis-data", "ocrd-import", "ocrd-make"]
```

Comment:
Why is this necessary? Why here?

AFAICS the cause of these false positives is simply the `ocrd-*` default glob …
https://github.com/OCR-D/core/blob/79c61e303c87f229d5c96aedc0da31ef82b0f5d3/src/ocrd/cli/resmgr.py#L48-L54
… combined with general search in PATH …
https://github.com/OCR-D/core/blob/79c61e303c87f229d5c96aedc0da31ef82b0f5d3/src/ocrd/resource_manager.py#L100-L104

So it would be more fitting to suppress these in resmgr.

Reply:
Right, I changed that.